api/src/common/decorators_api_views.py

```diff
@@ -0,0 +1,18 @@
+from django.utils.decorators import method_decorator
+from drf_yasg.utils import swagger_auto_schema
+
+# common decorators applied to all model view set classes
+def viewsets(tag):
```

Comment:
I'm not sure if we'll always only want 1 tag, since there may be times in the future when we want to assign multiple tags. Maybe this function should take in a list of tags instead?

Reply:
yeah changed the parameter to a list